File: php-pr/51/php/code-quality-error-handling/DatabaseConnector.php
<?php

class DatabaseConnector
{
    private $host = 'localhost';
    private $username = 'root';
    private $password = 'password';
    private $database = 'testdb';

    public function connect()
    {
        // {fact rule=code-quality-error-handling@v1.0 defects=1}
        try {
            $pdo = new PDO("mysql:host={$this->host};dbname={$this->database}", 
                          $this->username, $this->password);
            return $pdo;
        } catch (Exception $e) {
            // Empty catch block - swallows exceptions
        }
        // {/fact}
    }
}

File: ruby-pr/53/ruby/code-quality-error-handling/database_connector.rb
require 'sqlite3'

class DatabaseConnector
  def initialize
    @host = 'localhost'
    @database = 'testdb'
  end

  def connect
    # {fact rule=code-quality-error-handling@v1.0 defects=1}
    begin
      db = SQLite3::Database.new(@database)
      return db
    rescue => e
      # Empty rescue block - swallows exceptions
    end
    # {/fact}
  end
end